### PR TITLE
added neon-cli tests

### PR DESCRIPTION
**Pull request by kristinaNikolaevaa**

### CODE REVIEW

File: evm_loader/tests/test_cli.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+
+def test_help():
+    result = neon_cli().call("--help").strip()
+    assert len(result) > 1000
+
+
+def test_collect_treasury(evm_loader):
+    result = neon_cli().call(f"collect-treasury --evm_loader {evm_loader.loader_id}")
+    assert len(result) > 1000
```
Comment (gigimon) [inline evm_loader/tests/test_cli.py:97]: I think here we need add more logic to checks. Check addresses, transaction, balances, something that show us success executed command

File: evm_loader/tests/test_cli.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+
+
+def test_init_environment(evm_loader):
+    result = neon_cli().call(f"init-environment --evm_loader {evm_loader.loader_id}")
```
Comment (gigimon) [inline evm_loader/tests/test_cli.py:102]: same too

File: evm_loader/tests/test_cli.py [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+    assert len(result) > 1000
+
+
+def test_get_ether_account_data(evm_loader, user_account):
+    result = neon_cli().call(
+        f"get-ether-account-data --evm_loader {evm_loader.loader_id} {user_account.eth_address.hex()}")
```
Comment (gigimon) [inline evm_loader/tests/test_cli.py:108]: here we need to check result (account data)

File: evm_loader/tests/test_cli.py [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+def test_create_program_address(evm_loader):
+    seed = gen_hash_of_block(20)
+    result = neon_cli().call(
+        f"create-program-address --evm_loader {evm_loader.loader_id} {seed[2:]}").strip()
+    assert len(result.split(" ")[0]) == 44
+
+
+def test_deposit(evm_loader, user_account):
+    result = neon_cli().call(
+        f"deposit --evm_loader {evm_loader.loader_id} 10 {user_account.eth_address.hex()}").strip()
```
Comment (gigimon) [inline evm_loader/tests/test_cli.py:128]: maybe here we can check balance before/after?

**Review by gigimon — CHANGES_REQUESTED** [review]

File: evm_loader/tests/test_cli.py [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+    index = random.randint(0, 127)
+    treasury_pool_address = create_treasury_pool_address(index)
+    result = neon_cli().call(command_args)
+    assert f"{index}: skip account {treasury_pool_address}" in result
+
+    amount = random.randint(1, 1000)
+    trx = solana_client.request_airdrop(treasury_pool_address, amount)
+    wait_confirm_transaction(solana_client, trx['result'])
+    result = neon_cli().call(command_args)
+    assert f"{index}: collect {amount} lamports from {treasury_pool_address}" in result
```
Comment (gigimon) [inline evm_loader/tests/test_cli.py:108]: wait new amount in main pool address

**Review by gigimon — APPROVED** [review]

**Review by gigimon — APPROVED** [review]

File: evm_loader/tests/test_cli.py [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+
+def test_get_storage_at(evm_loader, operator_keypair, user_account, treasury_pool):
+    contract = deploy_contract(operator_keypair, user_account, "hello_world.binary", evm_loader, treasury_pool)
+    result = neon_cli().call(
+        f"get-storage-at --evm_loader {evm_loader.loader_id} {contract.eth_address.hex()} 0x0").strip()
+    assert len(result) > 500
+
+
+@pytest.mark.xfail(reason="https://neonlabs.atlassian.net/browse/NDEV-957")
+def test_cancel_trx(evm_loader, user_account, deployed_contract, operator_keypair, treasury_pool):
```
Comment (anton-lisanin) [inline evm_loader/tests/test_cli.py:166]: This test should pass now

**Review by anton-lisanin — APPROVED** [review]

**Pull request merged**